package.json

```diff
@@ -35,6 +35,7 @@
     "babel-eslint": ">=10.1.0",
     "eslint": ">=7.14.0",
     "eslint-plugin-prettier": ">=3.1.4",
+    "eslint-plugin-sort-imports-es6-autofix": ">=0.6.0",
```

Comment:
not sure if this one should exist in dependencies instead. I think it might be a requirement to ensure the consumer projects installs it.

Reply:
added it as a transitive one and pinned it at the latest minor/patch version. i guess it's the way to go if we don't want to maintain it in other repos. although we don't do the same for the other ones. thoughts?

Reply:
Makes sense to me. The reason that I kept other ones out of this approach is that we could still have an option to be able to update eslint, eslint-plugin-vue, prettier etc. on the consumer project by the need, without updating this repo. So it both gives some freedom in the consumer project, but also not maintaining this repo for each update. For this package I guess it does not require as much as maintaining as it is probably not updated as often as the other ones. WDYT?

Reply:
Personally, I prefer to handle all `eslint` plugins the same way for uniformity. But we can try this way and see.